app/src/main/res/layout/login.xml

```diff
@@ -40,10 +40,9 @@
         android:id="@+id/server_url"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentLeft="true"
-        android:layout_below="@+id/password"
         android:ems="10"
         android:hint="@string/server_url"
+        android:text="@string/api_zulip_com"
```

Comment:
This removes unused view layout attributes.  The parent view type changed from `RelativeLayout` to `LinearLayout`.  `LinearLayout` view groups don't use these types of child attributes to layout views.